src/login/LoginFailure.jsx

```diff
@@ -68,6 +70,24 @@ const LoginFailureMessage = (props) => {
       );
       break;
     }
+    case ALLOWED_DOMAIN_LOGIN_ERROR: {
+      const url = `${getConfig().LMS_BASE_URL}/dashboard/?tpa_hint=${context.tpa_hint}`;
+      const tpaLink = (
+        <a href={url}>
+          {context.provider}
```

Comment:
The error message has an underline on "<provide> account".
<img width="507" alt="Screen Shot 2022-09-22 at 11 58 48 PM" src="https://user-images.githubusercontent.com/40633976/191831617-318dd8db-6454-4e77-8457-39e119176027.png">